openid-data-classes/src/commonMain/kotlin/at/asitplus/iso/DeviceRequest.kt

```diff
@@ -1,8 +1,7 @@
 @file:OptIn(ExperimentalSerializationApi::class)
 
-package at.asitplus.wallet.lib.iso
+package at.asitplus.iso
```

Comment:
Please document those namespace changes, the move in general, and removal of the `serialize()` methods in the changelog